### ISSUE TITLE
Release of clustering models

### ISSUE THREAD

**Issue by mtran14** [facebookresearch/av_hubert#87]
Hello,

Thank you for sharing your work. 

Is it possible to release the k-means models that you use to generate the pseudo-labels for the final AVHuBERT-base and AVHuBERT-large from the released penultimate iteration models?

**Comment by prajwalkr** [comment]
Yes, please @authors, it would really help us to train new models and not be bottlenecked by the expensive k-means step.

**Comment by chevalierNoir** [comment]
Due to legal issues, releasing a new model like the k-means model will take some time. Usually training a k-means model from scratch following [the instructions](https://github.com/facebookresearch/av_hubert/tree/main/avhubert/clustering#k-means-clustering) doesn't require too much compute, and it should only take a few hours on a typical 128GB-RAM machine. Note we only use 10% of the whole training data for clustering. The fraction can be further reduced in large datasets like VoxCeleb2 with negligible impact on the performance.

**Comment by mtran14** [comment]
Thank you for your response. Limited computational resources for the k-means model is not really a problem for me. However, I am trying to adapt the released models on my dataset (aka task-adaptive pre-training) so fine-tuning the model with new pseudo-labels (from a k-means model trained from scratch) would result in lower performance.
